hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneTenantShell.java

```diff
@@ -523,4 +523,31 @@ public void testOzoneTenantBasicOperations() throws IOException {
     checkOutput(err, "Revoked accessId", false);
   }
 
-}
+  private void testListTenantUsers() throws IOException {
```

Comment:
what happens when we don't give tenant argument ? will it list all the kerberos principals ?

Reply:
The command mandates a "tenant" to be passed in. Else, it will fail.